.github/workflows/pr_check_go_ms.yml

```diff
@@ -2,23 +2,24 @@ name: Pull Request Checks
 
 on:
   workflow_call:
-    pull_request:
-      types: [opened, synchronize, reopened, ready_for_review]
-      branches:
```

Comment:
al quitar esta config, cual seria el trigger para que se ejecute el pieline?

Reply:
In theory, an implementation within the repository that integrates the workflow itself. I'm need to run a test first, but should work.

Reply:
I'm going to force the merge to test. This is a non breaking PR, so it's safe to do it.